### PR TITLE
Update `EveningStandard` parser

### PR DESCRIPTION
**Pull request by MaxDall**
This adjusts the `EveniingStandard` parser to the latest version.

### CODE REVIEW

File: src/fundus/publishers/uk/evening_standard.py [hunk 1]
```diff
@@ -40,3 +41,18 @@ def authors(self) -> List[str]:
         @attribute
         def topics(self) -> List[str]:
             return generic_topic_parsing(self.precomputed.meta.get("keywords"))
+
+    class V1_1(V1):
+        VALID_UNTIL = datetime.date.today()
+        _summary_selector = CSSSelector("div.sc-jgyXzG")
+        _subheadline_selector = CSSSelector("div#main div.sc-dFfFtc > h3")
```
Comment (addie9800) [inline src/fundus/publishers/uk/evening_standard.py:48]: It seems like not all articles use this new layout. [This article](https://www.standard.co.uk/lifestyle/julian-assange-wikileaks-life-stella-moris-marriage-relationships-b1167169.html) was published today and follows the old layout (which would also miss the subheadlines)

Comment (addie9800) [reply src/fundus/publishers/uk/evening_standard.py:48]: My guess are that some (but not all) of the lifestyle articles still use the old format for whatever reason: 
- https://www.standard.co.uk/lifestyle/wellness/modern-brain-clinic-dr-lesley-parkinson-clinic-adhd-depression-b1163587.html
- https://www.standard.co.uk/lifestyle/fashion/princess-diana-vivienne-westwood-elton-john-fashion-auction-boom-millions-b1167547.html

**Review by addie9800 — CHANGES_REQUESTED** [review]
Thanks for the update 👍

File: src/fundus/publishers/uk/evening_standard.py (hunk 1 above)
Comment (MaxDall) [reply src/fundus/publishers/uk/evening_standard.py:48]: Hmm, I would refrain from supporting both layouts. For now, I think it's best to drop support for `lifestyle` articles.

**Review by addie9800 — APPROVED** [review]
Sounds good

**Pull request merged**